sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -204,12 +201,12 @@ def test_make_recordset(client):
     exp = 'aia.lev1_euv_12s[2020.01.01_00:00:35_TAI-2020.01.01_01:00:35_TAI]'
     assert client._make_recordset(**d1) == exp
 
-    d1.update({'wavelength': 604*u.AA})
+    d1['wavelength'] = 604*u.AA
     exp = 'aia.lev1_euv_12s[2020.01.01_00:00:35_TAI-2020.01.01_01:00:35_TAI][604]'
     assert client._make_recordset(**d1) == exp
 
     del d1['wavelength']
-    d1.update({'primekey': {'WAVELNTH': '604'}})
+    d1['primekey'].update({'WAVELNTH': '604'})
```

Comment:
This is causing a real test fail because the 'primekey' key does not already exist

Reply:
Damn it. I will fix it.